mautic/mautic
# Remove unused translations

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:


The next translations are not used anywhere:
- mautic.salesforce.activity.points_added
- mautic.salesforce.activity.points_subtracted
- mautic.sugarcrm.activity.points_added
- mautic.sugarcrm.activity.points_subtracted

This PR offers to remove them

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Try to find where the translations are used in the code. There is no one line of code




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11329"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
